matlab/lpo_mex.cpp

```diff
@@ -328,51 +292,28 @@ std::shared_ptr<SeedFunction> createSeed( const std::string & s ) {
     }
     return seed;
 }
+static std::string model;
 static void newProposal( MEX_ARGS ) {
     if( nlhs == 0 ) {
         mexErrMsgTxt("newProposal expected one return argument");
         return;
     }
-    // Create an empty unary
-    ProposalSettings prop_settings;
-    prop_settings.unaries.clear();
-    
-    // Add all settings
+
     for ( int i=0; i<nrhs; i++ ) {
         std::string c = toString( prhs[i] );
-        
+
         // Process the command
-        if (c == "max_iou") {
-            if( i+1>=nrhs || !mxIsNumeric(prhs[i+1]) )
-                mexErrMsgTxt("max_iou numeric argument required");
-            prop_settings.max_iou = mxGetScalar(prhs[i+1]);
-            i++;
-        }
-        else if (c == "seed") {
+        if (c == "model") {
             if( i+1>=nrhs || !mxIsChar(prhs[i+1]) )
-                mexErrMsgTxt("seed string argument required");
-            prop_settings.foreground_seeds = createSeed( toString( prhs[i+1] ) );
+                mexErrMsgTxt("model argument required");
+            model =  toString( prhs[i+1] );
```

Comment:
Instead of saving a static string we might want to just load the model here and return it using
```
std::make_shared<LPO> p = std::make_shared<LPO>();
p->load(prhs[i+1]);
plhs[0] = ptr2mat( p );
```